src/components/Navbar/Navbar.jsx

```diff
@@ -1,15 +1,20 @@
 import "./navbar.css";
-
+import { useGlobal } from "../../context/GlobalContext";
 import { Link } from "react-router-dom";
 const Navbar = () => {
+  const { handleReset } = useGlobal();
   return (
     <>
       <nav className="nav-container flex-row">
         <div className="flex-row ">
           <Link to="/">
-            <h4 className="fw-500 nav-heading">
+            <h4 className="fw-500 nav-heading" onClick={() => handleReset()}>
```

Comment:
suggestion: no need to have an anonymous function, you can directly give the handler to the onClick, since you are not passing anything to that handler

Reply:
thanks for pointing out